tests/rkt_tests.go

```diff
@@ -133,19 +135,28 @@ func (ctx *rktRunCtx) dir(idx int) string {
 }
 
 func (ctx *rktRunCtx) reset() {
+	ctx.cleanupChildren()
 	ctx.runGC()
 	for _, d := range ctx.directories {
 		d.reset()
 	}
 }
 
+func (ctx *rktRunCtx) cleanupChildren() {
+	for _, child := range ctx.children {
+		log.Printf("Shutting down %q", child.Cmd.Path)
+		child.Cmd.Process.Kill()
```

Comment:
We could pass it on and then handle (ignore) it in the invoking function, would you prefer that?